tripy/tests/common/test_utils.py

```diff
@@ -60,33 +59,3 @@ def test_get_element_type():
 def convert_list_to_array(values, dtype, expected):
```

Comment:
Not part of your change, but this actually wouldn't have been running as a test because it's missing the `test_` prefix
```suggestion
def test_convert_list_to_array(values, dtype, expected):
```